### ISSUE TITLE
confused about the documentation, is the wrong endpoint suggested in the docs ?

### ISSUE THREAD

**Issue by pedro-nonfree** [systemli/prometheus-jitsi-meet-exporter#49]
I don't have much time to report on this, but I would like to have this discussion

I am surprised that in documentation it is suggested to use the jibri json endpoint

- https://github.com/systemli/prometheus-jitsi-meet-exporter#docker
- https://github.com/systemli/ansible-role-jitsi-meet-exporter#role-variables

but as seen in the code, the json endpoint is ready for the jitsi videobridge, which is activated when you add in `/etc/jitsi/videobridge/config`

```
JVB_OPTS="--apis=rest,"
```

then you have to restart jitsi videobridge `systemctl restart jitsi-videobridge2.service`

I also see in the code that wants the jitsi videobridge json endpoint

https://github.com/systemli/prometheus-jitsi-meet-exporter/blob/master/main.go#L16

so instead of looking for the jibri endpoint, it should point to the jitsi videobridge endpoint (when successfully configured as explained few lines above)

```diff
-docker run -p 9888:9888 systemli/prometheus-jitsi-meet-exporter:latest -videobridge-url http://jitsi:8888/stats 
+docker run -p 9888:9888 systemli/prometheus-jitsi-meet-exporter:latest -videobridge-url http://jitsi:8080/colibri/stats 
```

jibri json endpoint gives the following info:

```
# curl -s localhost:8888/stats | jq
{
  "xmpp_service": {
    "total_recv": 1049286,
    "total_sent": 349855
  },
  "largest_conference": 0,
  "conference_sizes": [
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0
  ],
  "total_conferences_created": 4,
  "threads": 221,
  "xmpp": {
    "total_recv": 1049286,
    "total_sent": 349855
  },
  "jingle": {
    "received": {
      "source-remove": 9,
      "transport-info": 5,
      "source-add": 8,
      "session-accept": 5
    },
    "sent": {
      "session-initiate": 5,
      "source-remove": 9,
      "source-add": 14
    }
  },
  "bridge_failures": {
    "participants_moved": 0,
    "bridges_removed": 0
  },
  "avg_allocate_channels_req_time_nanos": 0,
  "total_participants": 7,
  "participant_notifications": {
    "ice_failed": 0,
    "request_restart": 0
  },
  "bridge_selector": {
    "total_least_loaded_in_region": 0,
    "total_split_due_to_load": 0,
    "total_not_loaded_in_region_in_conference": 0,
    "in_shutdown_bridge_count": 0,
    "total_least_loaded_in_region_in_conference": 0,
    "total_not_loaded_in_region": 0,
    "total_split_due_to_region": 0,
    "bridge_count": 2,
    "operational_bridge_count": 2,
    "total_least_loaded_in_conference": 0,
    "total_least_loaded": 0
  },
  "jibri": {
    "total_sip_call_failures": 0,
    "live_streaming_pending": 0,
    "recording_pending": 0,
    "live_streaming_active": 0,
    "total_recording_failures": 0,
    "sip_call_pending": 0,
    "sip_call_active": 0,
    "total_live_streaming_failures": 0,
    "recording_active": 0
  },
  "conferences": 0,
  "participants": 0
}
```

jitsi videobridge gives the following info

```
# curl -s localhost:8080/colibri/stats | jq
{
  "inactive_endpoints": 0,
  "inactive_conferences": 0,
  "total_ice_succeeded_relayed": 0,
  "octo_version": 1,
  "total_loss_degraded_participant_seconds": 0,
  "bit_rate_download": 0,
  "local_active_endpoints": 0,
  "muc_clients_connected": 1,
  "total_participants": 0,
  "total_packets_received": 0,
  "rtt_aggregate": 0,
  "packet_rate_upload": 0,
  "p2p_conferences": 0,
  "total_loss_limited_participant_seconds": 0,
  "octo_send_bitrate": 0,
  "total_dominant_speaker_changes": 0,
  "receive_only_endpoints": 0,
  "total_colibri_web_socket_messages_received": 0,
  "octo_receive_bitrate": 0,
  "version": "2.1.492-g5edaf7dd",
  "total_ice_succeeded": 0,
  "total_colibri_web_socket_messages_sent": 0,
  "total_bytes_sent_octo": 0,
  "total_data_channel_messages_received": 0,
  "total_conference_seconds": 0,
  "num_eps_oversending": 0,
  "bit_rate_upload": 0,
  "total_conferences_completed": 0,
  "octo_conferences": 0,
  "num_eps_no_msg_transport_after_delay": 0,
  "region": "region1",
  "endpoints_sending_video": 0,
  "packet_rate_download": 0,
  "muc_clients_configured": 1,
  "outgoing_loss": 0,
  "overall_loss": 0,
  "conference_sizes": [
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0
  ],
  "total_packets_sent_octo": 0,
  "conferences_by_video_senders": [
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0
  ],
  "endpoints_with_high_outgoing_loss": 0,
  "stress_level": 0,
  "jitter_aggregate": 0,
  "total_ice_succeeded_tcp": 0,
  "octo_endpoints": 0,
  "current_timestamp": "2021-06-29 06:18:53.220",
  "total_packets_dropped_octo": 0,
  "conferences": 0,
  "participants": 0,
  "average_participant_stress": 0.01,
  "largest_conference": 0,
  "total_packets_sent": 0,
  "total_data_channel_messages_sent": 0,
  "incoming_loss": 0,
  "total_bytes_received_octo": 0,
  "octo_send_packet_rate": 0,
  "conferences_by_audio_senders": [
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0
  ],
  "total_conferences_created": 0,
  "total_ice_failed": 0,
  "threads": 37,
  "videochannels": 0,
  "total_packets_received_octo": 0,
  "graceful_shutdown": false,
  "octo_receive_packet_rate": 0,
  "total_bytes_received": 0,
  "total_loss_controlled_participant_seconds": 0,
  "total_partially_failed_conferences": 0,
  "endpoints_sending_audio": 0,
  "dtls_failed_endpoints": 0,
  "total_bytes_sent": 0,
  "mucs_configured": 1,
  "total_failed_conferences": 0,
  "mucs_joined": 1,
  "relay_id": "maskedip:4096"
}
```

**Comment by prayagsingh** [comment]
@pedro-nonfree The endpoint mentioned [here](https://github.com/systemli/prometheus-jitsi-meet-exporter#docker) is the `jicofo` endpoint and it's not correct. It should be jvb endpoint as mentioned by you. Below are the stats that the jicofo endpoint offers.

```
{
  "xmpp_service": {
    "total_recv": 16,
    "total_sent": 18
  },
  "jibri_detector": {
    "count": 0,
    "available": 0
  },
  "largest_conference": 0,
  "conference_sizes": [
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0
  ],
  "total_conferences_created": 0,
  "threads": 26,
  "xmpp": {
    "total_recv": 16,
    "total_sent": 18
  },
  "jingle": {
    "received": {},
    "sent": {}
  },
  "bridge_failures": {
    "participants_moved": 0,
    "bridges_removed": 0
  },
  "avg_allocate_channels_req_time_nanos": 0,
  "total_participants": 0,
  "participant_notifications": {
    "ice_failed": 0,
    "request_restart": 0
  },
  "queues": {},
  "jigasi": {},
  "bridge_selector": {
    "total_least_loaded_in_region": 0,
    "total_split_due_to_load": 0,
    "total_not_loaded_in_region_in_conference": 0,
    "in_shutdown_bridge_count": 0,
    "total_least_loaded_in_region_in_conference": 0,
    "total_not_loaded_in_region": 0,
    "total_split_due_to_region": 0,
    "bridge_count": 1,
    "operational_bridge_count": 1,
    "total_least_loaded_in_conference": 0,
    "total_least_loaded": 0
  },
  "jibri": {
    "total_sip_call_failures": 0,
    "live_streaming_pending": 0,
    "recording_pending": 0,
    "live_streaming_active": 0,
    "total_recording_failures": 0,
    "sip_call_pending": 0,
    "sip_call_active": 0,
    "total_live_streaming_failures": 0,
    "recording_active": 0
  },
  "conferences": 0,
  "participants": 0,
  "slow_health_check": 0
}
```
**Jibri endpoint**

`curl http://localhost:2222/jibri/api/v1.0/health`

***stats***

```
{
  "status": {
    "busyStatus": "IDLE",
    "health": {
      "healthStatus": "HEALTHY",
      "details": {}
    }
  }
}
```

**Comment by t2d** [comment]
Thanks for bringing this up. I believe this is a leftover from old configurations and I corrected it in 2f438c9.
